samples/sensor/tmp116/README.rst

```diff
@@ -0,0 +1,52 @@
+.. _ti_tmp116_sample:
+
+TI_TMP116 Sample
+################
+
+Description
+***********
+
+This sample application periodically takes Temperature using the ti_tmp116
+sensor driver. The result is written to the console.
+
+Requirements
+************
+
+This sample needs a TI TMP116 sensor connected to the target board's I2C
+connector.
+
+
+Wiring
+******
+
+This sample is tested with the STM32F401RE nucleo.
+
+The sensor operates at 3.3V and uses I2C to communicate with the board.
+
+External Wires:
+
+* Breakout **GND** pin <--> Nucleo **GND** pin
+* Breakout **VCC** pin <--> Nucleo **3V3** pin
+* Breakout **SDA** pin <--> Nucleo **CN5-D14** pin
+* Breakout **SCL** pin <--> Nucleo **CN5-D15** pin
+
+Building and Running
+********************
+
+This sample builds one application for the TMP 116 sensor.
+Build/Flash Steps:
```

Comment:
You should also mention connecting a USB cable between the development computer and the board (for flashing) and running a terminal emulator (e.g., minicom) and specify the settings, to see the console output.